README.rst

```diff
@@ -147,6 +147,24 @@ Fixtures
         """
         return {"service_log_path": Path("/log/directory/geckodriver.log")}
 
```

Comment:
Could you please add just a single line of text describing, what this code does, please? As in "The snippet below configures Chrome to ignore certificate errors and sets a specific window size". After that, I'm merging. Thanks.

Reply:
... also, if we only change docs, I'd love to see "[ci skip]" in the commit message, so CI won't get triggered. Like described here: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs . Thanks!